num2words/__init__.py

```diff
@@ -50,6 +51,7 @@
     'dk': lang_DK.Num2Word_DK(),
     'pt_BR': lang_PT_BR.Num2Word_PT_BR(),
     'he': lang_HE.Num2Word_HE()
```

Comment:
Sorry for the delay in the review.

This is invalid syntax. I suggest that you run [tox](https://pypi.python.org/pypi/tox) on the repo before pushing.